models/victoriametrics.go

```diff
@@ -0,0 +1,128 @@
+// pmm-managed
+// Copyright (C) 2017 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+package models
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strconv"
+
+	config "github.com/percona/promconfig"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	vmCacheDisableEnv = "VM_CACHE_DISABLE"
```

Comment:
Could you add explanatory comments for these ENV vars? Also it would be great if all these env vars consistently start with `VM_*` prefix.